docs/content/docs/tasks/restart-application-deployment/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Restart an Application Deployment
+description: Learn how to restart an unsuccessful Keptn Application Deployment .
+icon: concepts
+layout: quickstart
+weight: 20
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+## Restart an Application Deployment
+
+During the deployment of a `KeptnApp`, it might be that the deployment fails due to a unsuccessful pre-deployment evaluation or pre-deployment check.
+This might be because of, e.g., a misconfigured target value of a `KeptnEvaluationDefinition`, or of a wrong URL being checked in a pre deployment check.
```

Comment:
```suggestion
This could happen because of, e.g., a misconfigured target value of a `KeptnEvaluationDefinition`, or a wrong URL being checked in a pre deployment check.
```